File: oio337a/Algorithm_1/day_2/189.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        last = len(nums) - 1
        for i in range(k):
            poped_num = nums.pop(last)
            nums.insert(0, poped_num)

File: saewoo1/day2/189.java
class Solution {
    public void rotate(int[] nums, int k) {
        int[] arr = new int[nums.length];
        for (int i = 0; i < arr2.length; i++)
            arr[(i + k) % nums.length] = nums[i];
        for (int i = 0; i < nums.length; i++)
            nums[i] = arr[i];
		// char 배열로 변환 후에 reverse로 풀고싶었는데 지원 안해준대여
		// idx 변환은 똑같음
    }
}